.github/workflows/ci.yml

```diff
@@ -34,6 +34,8 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -r tools/requirements.txt
+    - name: check links
+      run: python tools/checkHeader.py
```

Comment:
It seems to be not very efficient. Consider add this as a step in `parseJson.py` in future PR?